src/app/containers/Disclaimer/index.test.jsx

```diff
@@ -26,7 +26,7 @@ const renderComponent = ({
   );
 
 describe('Disclaimer Component', () => {
-  it('Renders the text from the service config', () => {
+  it.skip('Renders the text from the service config', () => {
```

Comment:
Could make this test work by changing line 22 from this:

```
<ServiceContext.Provider value={{ disclaimer: { text: disclaimerText } }}>
```

to something like this I think:

```
<ServiceContext.Provider value={{ disclaimer: { block: [{ text: disclaimerText }] } }}>
```
```suggestion
  it.skip('Renders the text from the service config', () => {
```
The test on line 34 is also useless if we don't make this change

Reply:
Noticed this was fixed in the other PR

Reply:
Testing is done on the other a11y-commit PR :D